src/mobile-app/hooks/use-experiments.test.ts

```diff
@@ -47,7 +47,8 @@ describe("use-experiments hook", () => {
     const fetchMock = jest.fn().mockRejectedValue(new Error("test fetch failure"));
     (window as any).fetch = fetchMock;
 
-    const mockedStorage: IExperimentStorage = {
+    const mockedStorage = {
```

Comment:
Why did you remove the typing?  We lose compile time checks in the tests doing that.

Reply:
Ah, I see you changed to the LocalDataStorage.  I think you should create an interface that that class implements and then export it so that we can keep the type checking here.

Reply:
Right, because now it's a different class and this interface is gone. We get type checking when this object is passed to `useExperiments` a few lines below (as this hook is expecting specific type as an argument). That's why I was forced to add `localStorageKeyPrefix`. 
Adding a type here would be only helpful if this mock is never used I think. So, in practice not very useful. But I can add `const mockedStorage: LocalDataStorage<Experiments> =`. Or add a new interface that this storage implements that e.g. doesn't require `localStorageKeyPrefix`.